.gitignore

```
@@ -7,3 +7,8 @@ testing/
 testing*
 *.pyc
 null/
+/bin/hg38.fa
+/bin/hg38.fa.fai
+/bin/D2507392-ready.cram
+/bin/D2507392-ready.cram.crai
+/bin/test_output.bam
```

```suggestion
bin/hg38.fa
bin/hg38.fa.fai
bin/D2507392-ready.cram
bin/D2507392-ready.cram.crai
bin/test_output.bam
```
`/bin` is a root folder so it won't match these files

---

Can you also add all files that are not scripts to this ignore?